src/nameutil.rs

```diff
@@ -33,16 +35,15 @@ pub fn split_namespace_name(name: &str) -> (Option<&str>, &str) {
     (ns, name)
 }
 
-/* unused :(
-pub fn strip_suffix<'a>(name: &'a str, suffix: &str) -> Option<&'a str> {
-    if name.ends_with(suffix) {
-        Some(&name[..name.len() - suffix.len()])
-    }
-    else {
-        None
-    }
-}
-*/
+// unused :(
+// pub fn strip_suffix<'a>(name: &'a str, suffix: &str) -> Option<&'a str> {
+// if name.ends_with(suffix) {
+// Some(&name[..name.len() - suffix.len()])
+// }
+// else {
+// None
+// }
+// }
```

Comment:
(For this and other commented-out code): nobody seems to need it, why not just remove it?